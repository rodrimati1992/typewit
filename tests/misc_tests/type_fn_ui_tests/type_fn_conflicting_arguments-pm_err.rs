include!{"./type_fn_conflicting_arguments-err.rs"}