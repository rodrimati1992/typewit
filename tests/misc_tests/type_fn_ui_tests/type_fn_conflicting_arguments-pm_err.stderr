error[E0119]: conflicting implementations of trait `TypeFn<u8>` for type `Foo`
 --> tests/misc_tests/type_fn_ui_tests/./type_fn_conflicting_arguments-err.rs
  |
  |     impl u8  => u16;
  |          -- first implementation here
  |     impl<T> T => Vec<T>;
  |             ^ conflicting implementation for `Foo`
  |
  = note: this error originates in the macro `$crate::__impl_with_span` which comes from the expansion of the macro `typewit::type_fn` (in Nightly builds, run with -Z macro-backtrace for more info)
